server/src/api/resources/trip/trip.model.js

```diff
@@ -12,25 +12,25 @@ export const schema = {
     required: [true, "name is required."]
   },
   isArchived: {
-    type: boolean,
+    type: Boolean,
     default: false,
     required: [true]
   },
   start: {
-    type: ISOdate,
-    require: [true]
+    type: Date,
+    required: [true]
   },
   end: {
-    type: ISOdate,
-    require: [false]
+    type: Date,
+    required: [false]
   },
   lat: {
-    type: decimal,
-    require: [true]
+    type: Number,
+    required: [true]
   },
   lon: {
-    type: decimal,
-    require: [true]
+    type: Number,
+    required: [true]
   }
```

Comment:
@ahrjarrett Let’s go ahead and add a `waypoints` property right here. We’ll need it to establish a one-to-many relationship with `Waypoint`

```javascript 
waypoints: [{ type: ObjectId, ref: "Waypoint" }]
```